gocd/bci_repo_publish.py

```diff
@@ -190,7 +219,8 @@ def setup_tool(self):
 
         return tool
 
-    @cmdln.option('--token', help='The token for publishing. Does a dry run if not given.')
+    @cmdln.option('--repo-token', help='The token for publishing. Does a dry run if not given.')
```

Comment:
Nit picky, sorry. calling the token to trigger a release (or publish) repo-token is not very intuitive. Maybe release-token or publish-token (release being just accurate in OBS terminology)?

Reply:
Makes sense, I've changed it.